### PR TITLE
Remove paupers vault

### PR DESCRIPTION
**Pull request by darthbeep**
Also organized tools a little.

### CODE REVIEW

File: js/base-tool-animator.js [hunk 1]
```diff
@@ -1177,6 +1177,7 @@ function baseToolAnimator () {
 	d20plus.anim.animatorTool = {
 		name: "Token Animator",
 		desc: "Manage token animations",
+		mode: true,
```
Comment (TheGiddyLimit) [inline js/base-tool-animator.js]: `mode: true` is horribly undescriptive. Use e.g. `mode: "base"` instead, please

File: js/5etools-main.js [hunk 2]
```diff
@@ -3860,482 +3860,8 @@ To restore this functionality, press the "Bind Drag-n-Drop" button.<br>
...
-						});
-						table.save();
-						d20.rollabletables.refreshTablesList();
-						alert("Created table!")
-					}
-				});
-			},
-		},
-		{
-			name: "Pauper's Character Vault",
```
Comment (TheGiddyLimit) [inline js/5etools-main.js:3990]: What's the justification here? It already exists in VTTES..?
Please take the time to explain the _why_ when making PRs like this, either in the commit message (probably preferred in this case) or in a comment/the description of the PR

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]

File: js/5etools-main.js (hunk 2 above)
Comment (darthbeep) [reply js/5etools-main.js:3990]: Apologies for not stating, thought the trello counted. The pauper's vault is a duplicate feature of what VTTES offers.

File: js/base-tool-animator.js (hunk 1 above)
Comment (darthbeep) [reply js/base-tool-animator.js]: Fixed.

File: js/5etools-main.js (hunk 2 above)
Comment (TheGiddyLimit) [reply js/5etools-main.js:3990]: well there's the problem, I don't read the Trello 🙃 
Ideally, the PR should have a "key points" summary, and a link to the Trello where appropriate

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
On closer inspection, this `mode` stuff doesn't make sense--the 5etools-specific tools should be in a `5etools-tools.js` file, which can then be excluded from the build for the base script

**Comment by darthbeep** [comment]
Good point there, I'll fix it. Note that 5etools-tools.js doesn't yet exist, but should be easy enough to make.

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**